### PR TITLE
[Refactor] Remove some duplicate code, change some tiny mistakes in comment and log.

### PR DESCRIPTION
**Pull request by Linkerist**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [ ] enhancement
- [x] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

Remove some duplicate code/comment, change some tiny mistakes in code/comment when i read some code, no business logic changed, can be merged relatively easy.

### CODE REVIEW

File: be/src/storage/tablet_meta_manager.cpp [hunk 1]
```diff
@@ -386,8 +386,8 @@ Status TabletMetaManager::save(DataDir* store, const TabletMetaSharedPtr& tablet
     TabletMetaPB tablet_meta_pb;
     tablet_meta->to_meta_pb(&tablet_meta_pb);
     if (tablet_meta_pb.schema().keys_type() == KeysType::PRIMARY_KEYS) {
-        LOG(WARNING) << "does support save TabletMetaSharedPtr for PRIMARY_KEYS";
-        return Status::NotSupported("does support save TabletMetaSharedPtr for PRIMARY_KEYS");
+        LOG(WARNING) << "do not support save TabletMetaSharedPtr for PRIMARY_KEYS";
```
Comment (mofeiatwork) [inline be/src/storage/tablet_meta_manager.cpp:389]: `not support` ?

Comment (Linkerist) [reply be/src/storage/tablet_meta_manager.cpp:389]: yes.

File: be/src/runtime/plan_fragment_executor.cpp [hunk 2]
```diff
@@ -169,7 +169,7 @@ Status PlanFragmentExecutor::prepare(const TExecPlanFragmentParams& request) {
     profile()->add_child(_plan->runtime_profile(), true, nullptr);
     _rows_produced_counter = ADD_COUNTER(profile(), "RowsProduced", TUnit::UNIT);
 
-    VLOG(3) << "plan_root=\n" << _plan->debug_string();
+    VLOG(3) << "plan_root=" << _plan->debug_string();
```
[the remark this answers is not in the document]
Comment (Linkerist) [reply be/src/runtime/plan_fragment_executor.cpp]: yes

**Comment by Linkerist** [comment]
run starrocks_be_unittest

**Review by decster — APPROVED** [review]

**Review by chaoyli — APPROVED** [review]

**Pull request merged**